File: C/URI 1009.c
#include <stdio.h>
int main(){
  char NOME[50];
  double salario,montante,TOTAL;
  fgets(NOME,50,stdin);
  scanf("%lf %lf",&salario,&montante);
  TOTAL = salario + montante * 0.15;
  printf("TOTAL = R$ %.2lf\n",TOTAL);
  return 0;
}

File: Java/URI 1009.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      String nome;
      double salario, montante, comissao, total;
      nome = sc.nextLine();
      salario = sc.nextDouble();
      montante = sc.nextDouble();
      comissao = montante * (15.0/100.0);
      total = salario + comissao;
      System.out.printf("TOTAL = R$ %.2f%n", total);
      sc.close();
  }
}